website/templates/project/project_header.mako

```diff
@@ -77,7 +77,7 @@
                             <li><a href="${node['url']}addons/" class="subnav-header">Add-ons</a></li>
                         % endif
 
-                        % if user['has_read_permissions'] and not node['is_registration'] or (node['is_registration'] and permissions.WRITE in user['permissions']):
+                        % if user['has_read_permissions'] and not node['is_registration'] and not node['contributors'] or (node['is_registration'] and permissions.WRITE in user['permissions']):
```

Comment:
Update: I put a small `div` on the page `<div>node['contributors']=${node['contributors']}<div>` and confirmed that `node['contributors']` returns a list of contributors. For my local testing, it is

```
node['contributors']=[{'id': 'g573w', 'registered': True, 'surname': 'UVa', 'fullname': 'LC Test - UVa', 'shortname': 'LC Test - UVa', 'profile_image_url': 'https://secure.gravatar.com/avatar/0a7f82ac3c590cd6498ed4b5f769001b?d=identicon&s=40', 'active': True, 'visible': True, 'permission': 'admin', 'url': '/g573w/', 'absolute_url': 'http://localhost:5000/g573w/', 'display_absolute_url': 'localhost:5000/g573w', 'date_registered': '2023-01-13'}]
```

Reply:
Here are the screenshots:

* First three are for "Settings" disappeared from project pages

![Screen Shot 2023-03-02 at 1 33 31 AM](https://user-images.githubusercontent.com/3750414/222356121-25a73074-786d-4244-9774-274cb85ebb56.png)
![Screen Shot 2023-03-02 at 1 32 15 AM](https://user-images.githubusercontent.com/3750414/222356123-60730aa4-f7be-4503-9858-637559a77b17.png)
![Screen Shot 2023-03-02 at 1 28 15 AM](https://user-images.githubusercontent.com/3750414/222356124-8844e451-26c8-416e-93c5-a1908bc0b6e9.png)

* This is the one for what `node['contributors']` is

![Screen Shot 2023-03-02 at 1 46 46 AM](https://user-images.githubusercontent.com/3750414/222356486-cf1319aa-eec3-47c0-8088-e97dcd329011.png)

* Last one is the commit history

![Screen Shot 2023-03-02 at 1 51 35 AM](https://user-images.githubusercontent.com/3750414/222356613-f07387af-d34f-4326-a867-f7c39614ce04.png)

Reply:
Here is the update after I got my local env to work with registrations.

* As I've mentioned, I confirmed that your fix does't remove the "Settings" option from the contributors page when node is a registration.

![Screen Shot 2023-03-02 at 11 59 14 AM](https://user-images.githubusercontent.com/3750414/222499627-2f4bf2a8-cf97-4693-8320-ee41a4df1fd4.png)

* Here is the slack thread where I requested for help, which documents how I fixed my local env. Hope it helps in case you run into the same issue too: https://centerforopenscience.slack.com/archives/C03TPBCUC0K/p1677770676563329

Consider first pass CR done 🔥

Reply:
Thank you Longze, let me take a look at these.

Reply:
This has been resolved.